src/Shared/GatewaySurchargeHandler.php

```diff
@@ -5,11 +5,12 @@
 namespace Mollie\WooCommerce\Shared;
 
 use Mollie\WooCommerce\Gateway\Surcharge;
+use WC_Order;
 use \WC_Order_Item_Fee;
 
 class GatewaySurchargeHandler
 {
-    public $gatewayFeeLabel;
+    protected $gatewayFeeLabel;
```

Comment:
This breaks a test, though looks like it is not used outside of class anywhere else.